crypto/http/http_lib.c

```diff
@@ -8,7 +8,8 @@
  */
 
 #include <openssl/http.h>
-#include <openssl/httperr.h>
+#include <openssl/httperr.h> /* for BIO_snprintf() */
+#include <openssl/bio.h>
```

Comment:
The comment is a bit misplaced :wink:

Reply:
Looks like I should give up for the day :)